<launch>
  <node pkg="bachelor" type="TimerNode" name="TimerNode" output="screen" launch-prefix="gnome-terminal --command"/>
  <node pkg="bachelor" type="VideoPlayerNode" name="VideoPlayerNode" launch-prefix="gnome-terminal --command"/>
  <node pkg="bachelor" type="ObjectDetectorNode" name="ObjectDetectorNode" launch-prefix="gnome-terminal --command"/>
  <node pkg="bachelor" type="DisplayNode" name="DisplayNode" launch-prefix="gnome-terminal --command"/>
</launch>
